DataManagementSystem/Utilities/lfc_dfc_db_copy.py

```diff
@@ -577,56 +586,54 @@ def loadDataInMySQL( queryQueue, workerId ):
     # commit all, set back the foreign key check
     # close the connections, and bye bye
     if next_query is None:
-      cur.execute( "COMMIT;" )
-      cur.execute( "SET FOREIGN_KEY_CHECKS = 1;" )
+      cur.execute("COMMIT;")
+      cur.execute("SET FOREIGN_KEY_CHECKS = 1;")
       con.commit()
-      con.autocommit( True )
+      con.autocommit(True)
       cur.close()
       con.close()
       print("loadDataInMySQL EXITING ")
-      logfile.write( "loadDataInMySQL EXITING " )
+      logfile.write("loadDataInMySQL EXITING ")
       logfile.close()
       return
 
     # If we reach the threshold, we commit
     if queryExecuted % commitThreshold == 0:
-      cur.execute( "COMMIT;" )
+      cur.execute("COMMIT;")
       con.commit()
-      cur.execute( "START TRANSACTION;" )
+      cur.execute("START TRANSACTION;")
 
     # We try to execute the query
     # If ever it fail for a reason or another, we try again
     # If it fails gain, we ignore this query
     try:
-      cur.execute( next_query )
+      cur.execute(next_query)
     except Exception as e:
       print("worker %s : EXCEPTION %s\nworker %s :QUERY %s" % (workerId, e, workerId, next_query))
       print("worker %s trying again " % workerId)
 
-      logfile.write( "worker %s : EXCEPTION %s\nworker %s :QUERY %s" % ( workerId, e, workerId, next_query ) )
-      logfile.write( "worker %s trying again " % workerId )
+      logfile.write("worker %s : EXCEPTION %s\nworker %s :QUERY %s" % (workerId, e, workerId, next_query))
+      logfile.write("worker %s trying again " % workerId)
 
       try:
-        cur.execute( next_query )
-      except Exception, ee:
+        cur.execute(next_query)
+      except Exception as ee:
         print("worker %s COMPLETELY FAILED " % workerId)
         print("worker %s : EXCEPTION %s\nworker %s :QUERY %s" % (workerId, e, workerId, next_query))
```

Comment:
Is this `e` a mistake, and it should  be `ee`?

Reply:
I don't want to put any effort in this specific file.